task/upload.go

```diff
@@ -164,27 +163,14 @@ func getFileUrlForUploadTask(os *api.ObjectStore, cfg ImportTaskConfig, params a
 		u.Path = path.Join(u.Path, params.UploadedObjectKey)
 		return u.String(), nil
 	}
-	return getFileUrl(cfg, params.URL)
+	return getFileUrl(params.URL)
 }
 
-func getFileUrl(cfg ImportTaskConfig, url string) (string, error) {
+func getFileUrl(url string) (string, error) {
```

Comment:
I wonder if we even need this function now. WDYT of moving this error checking onto `getFileUrlForUploadTask` and deleting this helper? In `TaskTranscode` we can let it pass through an error later, it is not an expected case at all anyway (the API would already block that).

Reply:
Was not sure if you intended to extend it in the future so I left it. Will remove it and refactor around :+1: